alchemiscale/compute/api.py

```diff
@@ -187,13 +190,91 @@ def claim_taskhub_tasks(
         taskhub=taskhub_scoped_key,
         compute_service_id=ComputeServiceID(compute_service_id),
         count=count,
+        protocols=protocols,
```

Comment:
Note that `n4js.claim_taskhub_tasks` has protocols typed as `Protocol`s, not `str`.

Reply:
Thanks for your fixes in #279!